src/libs/actions/Session/index.js

```diff
@@ -289,6 +289,43 @@ function resetPassword() {
     API.write('RequestPasswordReset', {
         email: credentials.login,
     },
+    {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.ACCOUNT,
+                value: {
+                    errors: null,
+                    forgotPassword: true,
+                    message: null,
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.ACCOUNT,
+                value: {
+                    isLoading: false,
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.ACCOUNT,
+                value: {
+                    isLoading: false,
+                },
+            },
+        ],
```

Comment:
Hmm now that we don't set `isLoading` in optimisticData, there's no need to set it to false here. So we can just get rid of both `successData` and `failureData`